src/blueprint/blueprint.rs

```diff
@@ -219,4 +221,53 @@ impl Blueprint {
     // generation of schema cannot fail.
     schema.finish().unwrap()
   }
+
+  /// Generate a dumb schema useful for validation.
+  pub fn to_validation_schema(&self) -> Schema {
+    create(self, Some(SchemaModifiers { no_resolver: true }))
+      .validation_mode(ValidationMode::Strict)
+      .disable_introspection()
+      .finish()
+      .unwrap()
```

Comment:
prefer to return Result here instead of unwrap and make a validation error from this at the caller level